edweiss-app/app/(app)/lectures/slides/index.tsx

```diff
@@ -126,14 +139,15 @@ const LectureScreen: ApplicationRoute = () => {
             horizontal
             style={{
                 flex: 1,
+                backgroundColor: colorScheme == "dark" ? "#181825" : "#e6e9ef",
                 width: Dimensions.get('window').width * widthPorp,
                 height: Dimensions.get('window').height * heightProp,
             }}
         />
     );
 
     const ControlButtons = () => (
-        <TView alignItems='center' flexDirection='row' justifyContent='space-between' style={{ position: 'absolute', bottom: 0, left: 0, width: '100%' }} backgroundColor='overlay0'>
+        <TView alignItems='center' flexDirection='row' justifyContent='space-between' style={{ position: 'absolute', backgroundColor: colorScheme == "dark" ? "rgba(108, 112, 134, 0.5)" : "rgba(156, 160, 176, 0.5)", bottom: 0, left: 0, width: '100%' }}>
```

Comment:
So this is how you manage to make the `ControlButtons` transparent. Playing with the alpha values, makes more sens. I liked the way you handle the `light` and `dark` theme version.

Reply:
Yes thank you I wanted this overlay to be semi-transparent but we didn't have any constant values for it so I had to make them myself